Sources/MapboxNavigation/FeedbackSubtypeViewController.swift

```diff
@@ -14,9 +14,10 @@ class FeedbackSubtypeViewController: FeedbackViewController {
     /**
      Initialize a new FeedbackSubtypeViewController from a `NavigationEventsManager`.
      */
-    public init(eventsManager: NavigationEventsManager, feedbackType: FeedbackType) {
+    public init(eventsManager: NavigationEventsManager, feedbackType: FeedbackType, feedback: FeedbackEvent) {
```

Comment:
Let’s note this renamed public initializer in the changelog as a backwards-incompatible change.

Reply:
Actually this VC controller is internal, not sure why this init was marked as public.

Reply:
You can have a type to be internal in swift, and all methods marked as public will still be internal.
When you change the type from internal to the public, all public methods become public, and all internal methods stay internal. 
It might be non-obvious, as we see here. Maybe this is the result of refactoring a type from public to internal.

Reply:
If I remember correctly `FeedbackSubtypeViewController` should be exposed publicly, as it can be created from `FeedbackViewController`, which is public.

Reply:
It's not exposed publicly right now for sure, and I don't feel like marking it as public because anyway it can't be used without `FeedbackViewController`. I will remove these misleading `public` keywords from functions.